app/controllers/check_ins_controller.rb

```diff
@@ -0,0 +1,32 @@
+class CheckInsController < ApplicationController
+  # before_action :setup_variables, only: [:new, :create]
+  def index
+    @check_in = CheckInDecorator.new(current_account.filter(check_in_params))
+  end
+
+  def create
+    if @check_in.save
```

Comment:
There isn't a variable defined yet named `@check_in`, so you need to do something like `@check_in = CheckIn.new(check_in_params)` before you do `if @check_in.save`